WordPress/WordPressUITests/WordPressUITests.xctestplan

```diff
@@ -26,7 +26,8 @@
         "LoginTests\/testSelfHostedUsernamePasswordLoginLogout()",
         "LoginTests\/testUnsuccessfulLogin()",
         "LoginTests\/testWpcomUsernamePasswordLogin()",
-        "SignupTests\/testEmailSignup()"
+        "SignupTests\/testEmailSignup()",
+        "SupportScreenTests"
```

Comment:
Is it tracked somewhere that this test was (temporarily, I assume) disabled? (I'm assuming that was done because this test didn't pass but fixing it was a separate task for a separate PR… that needs a tracking issue to not forget about it)

Reply:
@thehenrybyrd beat me to merging this in https://github.com/wordpress-mobile/WordPress-iOS/pull/17015 – she'll take care of sorting it out! 🙂

Reply:
Also I've rebased on `develop` to fix the conflict associated with that :)